Cargo.toml

```diff
@@ -60,6 +60,7 @@ members = [
   "token/transfer-hook/cli",
   "token/transfer-hook/example",
   "token/transfer-hook/interface",
+  "token/confidential-transfer/proofs",
```

Comment:
are there going to be other confidential transfer crates? if so, this is fine, but if not, can we just put this in `token/confidential-transfer-proofs`?

Reply:
Yeah the plan was to also have the encryption crate for the ciphertext ops. The `encryption` module in the proofs crate will then refactor into that crate.